frontend/app/routes/$lang/_protected/personal-information/email-address/edit.tsx

```diff
@@ -163,7 +166,16 @@ export default function EmailAddressEdit() {
       <fetcher.Form method="post" noValidate>
         <input type="hidden" name="_csrf" value={csrfToken} />
         <div className="my-6">
-          <InputField id="emailAddress" name="emailAddress" type="tel" label={t('personal-information:email-address.edit.component.email')} required defaultValue={defaultValues.emailAddress} errorMessage={errorMessages.emailAddress} />
+          <InputField
+            id="emailAddress"
+            name="emailAddress"
+            type="tel"
+            label={t('personal-information:email-address.edit.component.email')}
+            required
+            defaultValue={defaultValues.emailAddress}
+            errorMessage={errorMessages.emailAddress}
+            disabled={isSubmitting}
```

Comment:
No need to disable the form inputs - only disable the buttons when the form is submitting.

(Same comment for other places where inputs are disabled).